ReactWindows/ReactNative.Shared/Views/View/ReactViewControl.cs

```diff
@@ -1,24 +1,37 @@
-﻿using System;
+using System;
 
 #if WINDOWS_UWP
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Data;
 #else
 using System.Windows;
 using System.Windows.Controls;
 #endif
 
-namespace ReactNative.UIManager
+namespace ReactNative.Views.View
 {
     /// <summary>
-    /// Represents a Canvas with an optional Border inside.
+    /// Native control for React view.
     /// </summary>
-    public class BorderedCanvas : Canvas
+    public class ReactViewControl : UserControl
```

Comment:
>UserControl [](start = 36, length = 11)

How did you decide to subclass `UserControl` instead of `ContentControl` or some other `Control` subclass? I don't have in-depth knowledge of XAML so I'm curious.

Reply:
Good question, honestly, I had originally created `ReactViewControl` from the Visual Studio User Control template, which is the only XAML template other than the Blank Page template, so that's what I went for (i.e., there's no `ContentControl` Add New Item template in Visual Studio). One thing that's intriguing about `UserControl` is that it sets itself as the routed event source for events, so it may be a slight optimization that clicks that occur on the inner Canvas will actually be "sourced" from the control (which is the "unit" that React cares about)